WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -352,61 +362,87 @@ public CoinListViewModel()
 						break;
 				}
 			});
+
+			InitList = ReactiveCommand.CreateFromTask(async () =>
+		   {
+			   // We have to wait for the UI to became visible to the user.
+			   await Task.Delay(1000);
```

Comment:
If in your CommandOnVisibleBehavior you used OnTemplatedApplied, do you still need this?

Reply:
Tested it and it is before the tab is shown.